dist/react-apiembed.js

```diff
@@ -1054,19 +1063,21 @@
 	                "li",
 	                {
 	                  role: "presentation",
-	                  className: "tabs-component-tab" + (index == _this2.state.active ? " is-active" : ""),
+	                  className: "tabs-component-tab" + (uniqueId + index == _this2.state.active ? " is-active" : ""),
 	                  key: index
 	                },
 	                React.createElement(
 	                  "a",
 	                  {
-	                    "aria-controls": "" + key,
+	                    "aria-controls": "" + (key + uniqueId),
 	                    "aria-selected": "true",
 	                    role: "tab",
 	                    className: "tabs-component-tab-a",
-	                    id: index,
-	                    href: "#" + key,
-	                    onClick: _this2.clickHandler
+	                    id: uniqueId + index,
+	                    href: "#" + (key + uniqueId),
```

Comment:
I know the `href` was being set before, but the resulting URL in the browser doesn't open to the correct snippet tab when you refresh / share the link. Additionally, it makes the page scroll jump when you click the tabs and hides the top of the snippet container. I think we should just remove the `href` attribute.